include/swift/AST/DiagnosticsCommon.h

```diff
@@ -49,7 +59,14 @@ namespace swift {
 #define DIAG(KIND,ID,Options,Text,Signature) \
     extern detail::DiagWithArguments<void Signature>::type ID;
 #include "DiagnosticsCommon.def"
-  }
-}
+  } // end namespace diag
+
+  namespace fixIt {
```

Comment:
Nitpick: namespaces are typically all-lowercase (cf. "namelookup") or lower_snake_case (cf. "ast_scope"). We do have some lowerCamelCase ones too but I think we're supposed to stick to this.